pkg/operator/apiservicecontroller/apiservice_controller.go

```diff
@@ -230,3 +261,106 @@ func (c *APIServiceController) eventHandler() cache.ResourceEventHandler {
 		DeleteFunc: func(obj interface{}) { c.queue.Add(workQueueKey) },
 	}
 }
+
+// getAPIServicesToManage gets the desired list of API Services that will be managed by this operator
+// note that some services might be managed by an external operators/servers
+func (c *APIServiceController) getAPIServicesToManage() ([]*apiregistrationv1.APIService, error) {
```

Comment:
make this function the one to pass through?  Doing so allows simple things to be simple and makes hard things possible.